tasks/instagrampoll.php

```diff
@@ -60,134 +60,137 @@ public function cleanup()
 
 	public function subscriptions()
 	{
-		$account = \Propeller\Instagram\Model_Account::query()
+		try 
+		{
+			$account = \Propeller\Instagram\Model_Account::query()
 				->where('active', 1)
 				->get_one();
 
-		$instagram = new \Instagram\Instagram;
-		$instagram->setAccessToken($account->token);
-
-		$subscriptions = \Propeller\Instagram\Model_Subscription::query()
-			->where('status', 'Live')
-			->get();
+			$instagram = new \Instagram\Instagram;
+			$instagram->setAccessToken($account->token);
 
-		foreach($subscriptions as $sub) {
+			$subscriptions = \Propeller\Instagram\Model_Subscription::query()
+				->where('status', 'Live')
+				->get();
 
-			$tag = $instagram->getTag($sub->object_id);
-			$media = $tag->getMedia();
-			$count = 0;
+			foreach($subscriptions as $sub) 
+			{
```

Comment:
Could we keep all braces for anything other than functions and classes on the same line please?